src/test/ui/lint/rfc-2383-lint-reason/expect_with_forbid.rs

```diff
@@ -0,0 +1,35 @@
+#![feature(lint_reasons)]
+
+#[forbid(unused_variables)]
+//~^ NOTE `forbid` level set here
+//~| NOTE `forbid` level set here
+#[expect(unused_variables)]
```

Comment:
Could you add another test that has the attributes in the order `#[expect] #[forbid]` ?

Reply:
The tests have been added. Then it simply works, like it would for `#[allow]`. The output also contains an `unfulfilled_lint_expectation` message which is expected as the forbidden diagnostic doesn't fulfill the expectation.